homeassistant/data_entry_flow.py

```diff
@@ -324,9 +324,13 @@ async def async_configure(
 
             # If the result has changed from last result, fire event to update
             # the frontend.
-            if (
-                cur_step["step_id"] != result.get("step_id")
-                or result["type"] == FlowResultType.SHOW_PROGRESS
+            if cur_step["step_id"] != result.get("step_id") or (
```

Comment:
These are rather critical parts of Home Assistant. Please add tests for this.

Thanks! 👍 

../Frenck

Reply:
A test has been updated to test this change.